### DISCUSSION TITLE
New to zeek

### DISCUSSION THREAD

**Discussion by djsnider** [zeek/zeek#1745] [How-Tos]
Good afternoon. I am looking to get insight on the initial setup and configuration. Any help is greatly appreciated.

**Comment by ckreibich** [comment]
Welcome! Please take a look at our [docs](https://docs.zeek.org/en/v4.1.0/) and swing by the Zeek Slack or the mailing list for assistance. Details here: https://zeek.org/community/ — thanks!

**Reply by djsnider** [reply]
Thank you for the reply. I have it installed and continue running into an error trying to run zeekctl

**Comment by G0stik** [comment]
Hello , I have a kind of difficult time to install zeek on kali, ive installed all the dependencies, zeek was installed in user/bin/zeek so i guess it was added to the path as it won't let me put it in manully, still can't run it. Does anybody know where could be a problem please ?

**Reply by 0xxon** [reply]
Hi - as Christian mentioned in the [reply above](https://github.com/zeek/zeek/discussions/1745#discussioncomment-1235200) - please look at our documentation, or join our community resources to get help. Github issues/discussions are seen by very few people, and we don't use them for support questions.
